pkg/inject/config.go

```diff
@@ -166,8 +165,7 @@ func CreateInjectionConfig(
 
 	// if not set, agent will get it from NATS
 	if mtlsModeAnnotation != "" {
-		modeErr := ValidateMTLSAnnotation(mtlsModeAnnotation, meshConfig.Mtls.Mode)
-		if modeErr != nil {
```

Comment:
Can this be renamed to `err`, or does the linter complain about shadowing?

Reply:
The linter did not give an error. I have renamed it to `err`.